### ISSUE TITLE
Depositing & withdrawing from town coffers.

### ISSUE THREAD

**Issue by Talesu** [WiIIiam278/HuskTowns#159]
Currently it's only possible to deposit money into your town bank. It seems logical there would also be a command that would allow players to take the money out of t he town coffers.

**Comment by WiIIiam278** [comment]
Not really. The reason for depositing into the town coffers is to level up your town -- what would withdrawing do, "level down" the town? I would like to expand this featureset in the future to possibly accomodate this, but that's already being tracked as per the roadmap accessible through the plugin docs.
